app.js

```diff
@@ -41,21 +41,17 @@ if(!redisService || redisService == null) {
     credentials = {"hostname":"127.0.0.1", "port":6379};
   }
 } else {
-  if(isDocker) {
-    //This works around a problem with networking when deployed to Bluemix in a docker
-    //container
-    //For some reason it takes about 30 seconds for the networking to come up on the container
-    //so we sleep here before we continue on and use these credentials to connect
-    console.log('The app is running in a Docker container on Bluemix so we are ' +
-      'sleeping for 90 seconds waiting for the networking to become active.');
-    require('sleep').sleep(90);
-  }
+  console.log('The app is running in a Docker container on Bluemix.')
```

Comment:
You would want to surround this log statement in if(isDocker) since you could be running as a CF app as well